spring-integration-jdbc/src/main/java/org/springframework/integration/jdbc/store/JdbcChannelMessageStore.java

```diff
@@ -86,7 +86,8 @@
  * </p>
  * <p>
  * Contrary to the {@link JdbcMessageStore}, this implementation uses one single
- * database table only. The SQL scripts to create the necessary table are packaged
+ * database table only and optimized for queue behaviour.
```

Comment:
Maybe: 'this implementation uses a single database table, optimized to operate like a queue'. I think it would help to explain what optimizations are made, if we can keep it brief.